dfsclient/src/test/resources/child-index.ecl

```diff
@@ -0,0 +1,25 @@
+childRec := RECORD
```

Comment:
at the risk of being unnecessarily verbose, let's ensure the file name is self documenting...
generate-nested-dataset-and-index.ecl?